apps/emqx_authn/src/simple_authn/emqx_authn_mongodb.erl

```diff
@@ -146,19 +146,19 @@ authenticate(#{password := Password} = Credential,
     case emqx_resource:query(Unique, {find_one, Collection, Selector2, #{}}) of
         undefined -> ignore;
         {error, Reason} ->
-            ?SLOG(error, #{msg => "query failed",
-                           unique => Unique,
+            ?SLOG(error, #{msg => "mongodb_query_failed",
+                           resource => Unique,
                            reason => Reason}),
             ignore;
         Doc ->
             case check_password(Password, Doc, State) of
                 ok ->
                     {ok, #{is_superuser => is_superuser(Doc, State)}};
                 {error, {cannot_find_password_hash_field, PasswordHashField}} ->
-                    ?SLOG(error, #{msg => "can't find password hash field",
-                                   unique => Unique,
+                    ?SLOG(error, #{msg => "cannot_find_password_hash_field",
+                                   resource => Unique,
                                    password_hash_field => PasswordHashField}),
-                    {error, bad_username_or_password};
+                    ignore;
```

Comment:
why `ignore` here ?
if if mongodb is configured to be used as auth provider, why should be a missing column error be ignored ?

Reply:
Since the fields in the MongoDB document are not fixed at all times, this error may occasionally appear at runtime. I don't want the user to have multiple authentication instances because this error causes another instance to fail to work.